src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.UniversalDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    private static final Scanner scanner = new Scanner(System.in);
+    @Inject
+    private UniversalDao<Bet> betUniversalDao;
```

Comment:
generic types get erased at runtime
this field becomes just `UniversalDao betUniversalDao`
as well as `UniversalDao<User>` becomes `UniversalDao`

let's not use UniversalDao and create two separate interfaces instead - BetDao and UserDao